### DISCUSSION TITLE
With the latest version facing issue with Jackson-databind

### DISCUSSION THREAD

**Discussion by rajeevrawat982** [networknt/json-schema-validator#799] [Q&A]
In my project multiple dependency bringing old version of jackson-databind (2.11.3) because of that I'm keep getting exception 

java.lang.NoSuchMethodError: 'boolean com.fasterxml.jackson.databind.JsonNode.canConvertToExactIntegral()'  ,

 as per my knowledge this method  was introduced in a newer version of the Jackson library (version 2.12.0).  Is there any work around so I can resolve this without upgrading the jackson-databind version ?

**Comment by stevehu** [comment]
You can explicitly add Jackson dependencies in your pom.xml

**Reply by rajeevrawat982** [reply]
After adding explicitly it is breaking other aws dependencies in project
